packages/target-tauri/src-tauri/src/help_window.rs

```diff
@@ -93,12 +101,13 @@ pub(crate) async fn open_help_window(
     {
         let _ = set_float_on_top_based_on_main_window(&help_window);
 
-        let help_window_clone = help_window.clone();
+        let help_window_arc = Arc::new(help_window);
+        let help_window_clone = Arc::clone(&help_window_arc);
         menu_manager
             .register_window(
                 &app,
-                &help_window,
-                Box::new(move |app| create_help_menu(app, &help_window_clone)),
+                &*help_window_arc,
+                Box::new(move |app| create_help_menu(app, &*help_window_clone)),
```

Comment:
OK, obeying the clippy warning `warning: deref which would be done by auto-deref` causes a compile error because our `WindowAbstraction` doesn't implement for `Arc<...`. I'm not sure what's the best way to fix this, besides adding `impl ... for Arc`.